submodules/usersCallForwarding/views/complexStrategy.html

```diff
@@ -0,0 +1,184 @@
+                                    {{/select}}
+                                </select>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div>
+                    {{#monsterRadio @root.i18n.users.callForwarding.strategies.common.phoneNumber.label }}
+                    <input type="radio" class="radio-state" name="{{strategy}}.type" value="phoneNumber" {{#compare
+                        type '===' 'phoneNumber' }} checked{{/compare}} />
+                    {{/monsterRadio}}
+
+                    <div class="selective-control-group phone-number disabled">
+                        <label for="{{strategy}}.phoneNumber.value" class="control-label">
+                            {{@root.i18n.users.callForwarding.strategies.common.phoneNumber.inputLabel}}
+                        </label>
+                        <div class="controls specific-controls">
+                            <input class="controlls-width" name="{{strategy}}.phoneNumber" type="phoneNumber" value="{{number}}">
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="active-duration">
+            <label>
+                Active duration
+            </label>
+
+            <div class="complex-rule">
+                {{#monsterRadio @root.i18n.users.callForwarding.strategies.common.length.always }}
+                <input type="radio" class="radio-state" name="{{strategy}}.length" value="always" checked="checked"
+                    {{#compare type '===' 'always' }} checked{{/compare}} />
+                {{/monsterRadio}}
+
+                {{#monsterRadio @root.i18n.users.callForwarding.strategies.common.length.custom }}
+                <input type="radio" class="radio-state" name="{{strategy}}.length" value="custom" {{#compare
+                    type '===' 'custom' }} checked{{/compare}} />
+                {{/monsterRadio}}
+
+                <div class="office-hours-wrapper disabled"></div>
+            </div>
+        </div>
+
+        <div class="remove-rule">
+            <label>
+                Remove rule
+            </label>
+
+            <div class="complex-rule">
+                <a class="remove-button">X</a>
+            </div>
+        </div>
+    </div>
+
+    <div class="options">
+        <div>
+            {{@root.i18n.users.callForwarding.strategies.common.options.title}}
+        </div>
+        <div class="options-container">
+
+            <div class="option" strategy="{{strategy}}">
+                <div class="options-title">
+                    {{@root.i18n.users.callForwarding.strategies.common.options.forward.title}}
+
+                    {{#monsterCheckbox "large-checkbox"}}
+                    <input type="checkbox" name="{{strategy}}.isEnabled" {{#if enabled}} checked="checked" {{/if}} />
+                    {{/monsterCheckbox}}
```

Comment:
Just the spacing here and in other places
```suggestion
                    {{#monsterCheckbox "large-checkbox"}}
                        <input type="checkbox" name="{{strategy}}.isEnabled" {{#if enabled}} checked="checked" {{/if}} />
                    {{/monsterCheckbox}}
```